drivers/include/si7021.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2016 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    driver_si7021 SI7021
+ * @ingroup     drivers
+ * @brief       Driver for the SI7021 temperature and humidity sensor.
+ * @{
+ *
+ * @file
+ * @brief       Interface definition of the SI7021 driver.
+ *
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ */
+
+#ifndef SI7021_H_
+#define SI7021_H_
+
+#include <stdint.h>
+
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name SI7021 address
+ */
+#define SI7021_ADDRESS              (0x80)
```

Comment:
Is this address always fixed? Or is there a way to re-programm it (through I2C commands or some external pin)?